doc/components.md

```diff
@@ -422,6 +422,99 @@ It is also possible to create custom renderers by extending the `ParallaxRendere
 Three example implementations can be found in the
 [examples directory](https://github.com/flame-engine/flame/tree/main/examples/lib/stories/parallax).
 
+## ShapeComponents
+
+There exists three basic components which can be used if you want to draw geometrical shapes as
+components on the screen. Since the `ShapeComponent`s are `PositionComponent`s you can use effects
+on them. All `ShapeComponent`s take a `Paint` as an argument and then arguments to define
+the shape of the specific component, it also takes all the arguments that can be passed to the
+`PositionComponent`.
+
+### CircleComponent
+A `CircleComponent` can be created only defining it's `radius`, but you most likely want to pass it
+a `position` and maybe `paint` (the default is white) too.
+
+Example:
+```dart
+final paint = BasicPalette.red.paint()..style = PaintingStyle.stroke;
+final circle = CircleComponent(radius: 200.0, position: Vector2(100, 200), paint: paint);
+```
+
+### RectangleComponent
+A `RectangleComponent` can be created in two ways, depending on if it is a square or not.
+To create a `RectangleComponent` that is 300px wide and 200px in height you can do the following:
```

Comment:
I don't have a strong opinion about this, but maybe we should refrain from using px on docs as Flutter uses logical pixels instead. Not sure if having mo suffix or just dpi would be better or more accurate

Reply:
Makes sense, fixed